### PR TITLE
[IDIS-1111] Fix for error changing password

### PR DESCRIPTION
**Pull request by dpoirier**
This was a more general problem that some URLs were only valid
with a prefix, and in some contexts like changing a password,
no prefix was known.

These changes:

* Make any URL that was valid with a prefix also valid with no prefix
* Modify the Debate model to allow an empty prefix

It might be necessary to always have a debate configured with no
prefix for everything to work... hopefully we can live with that
for now?

### CODE REVIEW

File: opendebates/models.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ class Debate(CachingMixin, models.Model):
     THEME_CHOICES = [(theme, theme) for theme in settings.SITE_THEMES]
 
     site = models.ForeignKey(Site, related_name='debates')
-    prefix = models.SlugField()
+    prefix = models.SlugField(blank=True, default='')
     theme = models.CharField(max_length=255, choices=THEME_CHOICES)
```
Comment (dpoirier) [inline opendebates/models.py:55]: Would it be worth the trouble to add a unique_together on site + prefix?

Comment (tobiasmcnulty) [reply opendebates/models.py:55]: That sounds like a good idea.

Comment (dpoirier) [reply opendebates/models.py:55]: Done.

**Comment by tobiasmcnulty** [comment]
The admin change password issue was fixed via #254. Some of the other changes in this PR we may want to revisit at some point, but probably as separate issues.

**Pull request closed without merging**